uber/models.py

```diff
@@ -1183,7 +1188,7 @@ def unregistered_badges(self):
 
     @cost_property
     def table_cost(self):
-        return sum(c.TABLE_PRICES[i] for i in range(1, 1 + int(self.tables)))
+        return sum(c.TABLE_PRICES[i] for i in range(1, 1 + int(float(self.tables))))
```

Comment:
Yeah, Numeric makes perfect sense. This line in particular will be fine, because ultimately we're casting it to an int anyway.